### PR TITLE
Add get apps route

### PR DESCRIPTION
**Pull request by oleh-momot**

### CODE REVIEW

File: registry/server/routes/apps/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default as apps } from './apps';
```
Comment (StyleT) [inline registry/server/routes/apps/index.ts]: registry/server/routes/apps/routes/getApps.ts -> registry/server/routes/apps/getApps.ts

File: registry/server/core/apps/repositories/selectApps.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import db from '../../../db';
```
Comment (StyleT) [inline registry/server/core/apps/repositories/selectApps.ts]: Remove for now. Move to route

File: registry/server/core/apps/services/reduceApps.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash/fp';
```
Comment (StyleT) [inline registry/server/core/apps/services/reduceApps.ts]: - Make it generic
- Pick more specific name
- Remove 1-2 line functions that you use only once

File: registry/server/core/apps/repositories/selectApps.ts (hunk 2 above)
Comment (oleh-momot) [reply registry/server/core/apps/repositories/selectApps.ts]: Removed

File: registry/server/core/apps/services/reduceApps.ts (hunk 3 above)
Comment (oleh-momot) [reply registry/server/core/apps/services/reduceApps.ts]: I created `postProcessResponse` for db
So it gives opportunities to process result of queries before they will be returned from db queries

File: registry/server/routes/apps/index.ts (hunk 1 above)
Comment (oleh-momot) [reply registry/server/routes/apps/index.ts]: Moved

File: registry/server/apps/interfaces/App.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  name: string,
+  spaBundle: string,
+  cssBundle: string,
+  dependencies: string,
+  ssr: string,
+  initProps: string,
+  props: string,
+  assetsDiscoveryUrl?: string,
+  assetsDiscoveryUpdatedAt: number,
+}
```
Comment (StyleT) [inline registry/server/apps/interfaces/App.ts]: Add newlines at the end of files.

**Pull request merged**